js/huobi.js

```diff
@@ -4785,7 +4785,7 @@ module.exports = class huobi extends Exchange {
         //     }
         //
         const data = this.safeValue (response, 'data', []);
-        const allAddresses = this.parseDepositAddresses (data, [ code ], false);
+        const allAddresses = this.parseDepositAddresses (data, [ currency['code'] ], false);
```

Comment:
@ttodua noticed something weird, the address returned is different depending on the format used (unified `USDT` or `usdt`)
![image](https://user-images.githubusercontent.com/43336371/220135781-5e2e14db-a401-45e9-af78-605669fa89cc.png)